apps/content_filters/filter_condition/filter_condition_field.py

```diff
@@ -205,3 +215,29 @@ def get_elastic_query(self):
 
     def get_mongo_query(self):
         return {'schedule_settings.utc_embargo': {'$gt': utcnow()}}
+
+
+class FilterConditionControlledVocabularyField(FilterConditionField):
+    def __init__(self, field):
+        self.field = type('_ControlledVocabulary', (object,), {'name': field})()
+        self.entity_name = 'subject.qcode'
+        self.field_type = str
+
+    def is_in_article(self, article):
+        return any([self.field.name == subject['scheme'] for subject in article.get('subject', [])])
```

Comment:
afaik there is no scheme set for default subjectcodes, so this would be keyerror